setup.py

```
@@ -1,15 +1,36 @@
+]
+if limited_api := not get_config_var("Py_GIL_DISABLED"):
+    macros.append(("Py_LIMITED_API", "0x030A0000"))
+
+if system() != "Windows":
+    cflags = ["-std=c11", "-fvisibility=hidden"]
+else:
+    cflags = ["/std:c11", "/utf-8"]
+
 
 class Build(build):
     def run(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `run`

Add return type annotation: `None`

(ANN201)